views/partials/footer.html

```diff
@@ -13,8 +13,18 @@
     </nav>
 
     <span class="footer-social">
-      <a class="footer-nav-item icon" href="https://github.com/electron" title="Github Organization"><i class="devicon-github-plain vertical-middle"></i></a>
-      <a class="footer-nav-item icon" href="https://twitter.com/electronjs" title="@electronjs on Twitter"><i class="devicon-twitter-plain vertical-middle"></i></a>
+      <a class="footer-nav-item" href="https://github.com/electron" aria-label="Github Organization">
```

Comment:
Quick note while I'm here, I noticed the `aria-label` on this link overrides the accessible name defined by the `<title>` in the SVG.